apps/storybook-react-native/stories/Backgrounds.stories.tsx

```diff
@@ -65,18 +70,16 @@ const TextComponents = () => (
         <Icon name={IconName.Info} color={IconColor.IconMuted} />
         <Text color={TextColor.TextMuted}>Text Muted and Icon Muted</Text>
       </Box>
+      <TextButton>Text Button</TextButton>
     </Box>
   </>
 );
 
 const Backgrounds: React.FC = () => {
   const tw = useTailwind();
   return (
-    <ScrollView style={tw`flex-1 bg-default`}>
```

Comment:
background/default is right up against the edge of text here. Going to move to change storybook background color altogether